### DISCUSSION TITLE
Rails engine like feature

### DISCUSSION THREAD

**Discussion by setoelkahfi** [loco-rs/loco#1051] [Ideas]
What I like about Rails is its pluggability, especially the [Rails engine](https://guides.rubyonrails.org/engines.html). Is Loco ready for that?

**Comment by jondot** [comment]
I have never thought about that but I think the answer is yes. 
loco apps can run inside other loco apps, it should be doable. 
now the question is which kinda use case do we want?

**Reply by setoelkahfi** [reply]
I'm thinking about the ActiveAdmin equivalent of Loco.
